backend/migrations/1729750071940-create-email-verification-tokens.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("email_verification_tokens", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      token: {
+        type: Sequelize.STRING,
+        allowNull: false,
```

Comment:
The 'token' field should be indexed to improve lookup performance, especially if this table will be queried frequently.